code/modules/reagents/machinery/chem_dispenser.dm

```diff
@@ -22,23 +21,23 @@
 	var/cell_charger_additon = 0 //This is not a TRUE/FALSE
 	var/accept_beaker = TRUE //At TRUE, ONLY accepts beakers.
 	var/hackedcheck = FALSE
-	var/list/dispensable_reagents
+	var/list/dispensable_reagents //I seriously hope this fixes the scrambling on part upgrade.
 	var/list/level0 = list(
-		"hydrazine","lithium","carbon",
-		"ammonia","acetone","sodium",
-		"aluminum","silicon","phosphorus",
-		"sulfur","hclacid","potassium",
-		"iron","copper","mercury",
-		"radium","water","ethanol",
-		"sugar","sacid","tungsten"
+		"acetone", "aluminum", "ammonia",
+		"carbon", "copper", "ethanol",
+		"hclacid", "hydrazine", "iron", 
+		"lithium", "mercury", "phosphorus",
+		"potassium", "radium", "sacid",
+		"silicon", "sodium", "sugar",
+		"sulfur", "tungsten", "water",
 	)
 
 	var/list/level1 = list("oil", "cryptobiolin")
 	var/list/level2 = list("toxin", "sodiumchloride")
-	var/list/level3 = list("potassium_chloride", "cryptobiolin")
+	var/list/level3 = list("mutagen")
```

Comment:
```suggestion
	var/list/level3 = list("mutagen", "potassium_chloride", "cryptobiolin") 
```
They unlock for a reason

Reply:
That cryptobiolin is a dupe, it needs to go.